### DISCUSSION TITLE
[Taskbar Clock Customization] support week number

### DISCUSSION THREAD

**Discussion by jerone** [ramensoftware/windhawk-mods#47] [Q&A] [answered]
How can I add an week number to Taskbar Clock Customization mod?

**Comment by m417z** [comment]
Not with the mod itself, since [the format](https://learn.microsoft.com/en-us/windows/win32/intl/day--month--year--and-era-format-pictures) doesn't support week numbers. Here's [a quick fork](https://gist.github.com/m417z/28819dfa10839b84b2acfa081a6429ec) which adds support to week numbers. Let me know if you need help installing it.

![image](https://user-images.githubusercontent.com/4129781/196769703-4aa9e2be-64b9-4455-918b-c65afba9f029.png)

**Comment by m417z** [comment, marked as answer]
Week numbers were added as %weeknum% in Taskbar Clock Customization v1.0.6.

**Reply by Gippo68** [reply]
Hi,
unfortunately, it shows the wrong number (one week less in my case).

**Reply by m417z** [reply]
The number is 1 for Sunday, 2 for Monday, ... 7 for Saturday. There's a [First day of week](https://stackoverflow.com/questions/39325212/how-get-start-day-of-week) setting in Windows, I'll look into using that to adjust the numbers to the user preference.

**Reply by Gippo68** [reply]
Thank you m417z,
my first-day setting is correct - Monday, today - but this is what it shows to me.
Many thanks for your help.

<img width="354" alt="Screenshot 2023-01-16 alle 18 33 09" src="https://user-images.githubusercontent.com/116503681/212737733-b8cceda4-3d20-4a2d-8338-58eb2b7f409a.png">

**Reply by m417z** [reply]
Done in Taskbar Clock Customization v1.0.7, now it respects the system configuration for the first day of the week.

**Comment by marcelser** [comment]
Did I got you right that %weeknum% is the day of week? normal people usually use the term "week number" for current week in year and the thing which is now %weeknum% is usually %dow%, Is there a possibility to show current week in year? Can you provide an update for this?
